controller/src/main/java/io/pravega/controller/server/ControllerServiceStarter.java

```diff
@@ -184,6 +177,23 @@ protected void startUp() {
                         .thenAcceptAsync(x -> controllerEventProcessors.startAsync(), controllerExecutor);
             }
 
+            // Setup and start controller cluster listener after all sweepers have been initialized.
+            if (serviceConfig.isControllerClusterListenerEnabled()) {
```

Comment:
there shouldnt be any.. the earlier dependency was removed by making cluster listener sweeper type agnostic.. and none of the sweepers depends on cluster listener.